code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -10,6 +10,7 @@
 	message = "cries."
 	message_mime = "sobs silently."
 	emote_type = EMOTE_AUDIBLE | EMOTE_VISIBLE
+	stat_allowed = SOFT_CRIT
```

Comment:
Does this still let you do it when you're conscious?

Reply:
yep, works perfectly 
when unconscious in crit:
![image](https://user-images.githubusercontent.com/78111117/205379667-a5d41fad-8915-464c-b722-72fc4587ea2d.png)

when conscious in crit:
![image](https://user-images.githubusercontent.com/78111117/205379780-c47ef06b-fe51-4dde-b832-779fb7925b39.png)